src/main/java/lotto/model/Game.java

```diff
@@ -0,0 +1,34 @@
+package lotto.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static lotto.common.Constants.*;
+
+public class Game {
+
+    private List<Integer> numbers;
+
+    public Game() {
+        this.numbers = generateNumbers();
+        System.out.println(this.numbers);
+    }
+
+    /**
+     * 난수목록 생성
+     *
+     * @return GET_NUMBER_COUNT 길이의 배열을 반환
+     */
+    private List<Integer> generateNumbers() {
+        List<Integer> randomNumbers = IntStream.range(MIN_RANGE_VALUE, MAX_RANGE_VALUE).boxed().collect(Collectors.toList());
+        Collections.shuffle(randomNumbers);
+        return randomNumbers.subList(0, GET_NUMBER_COUNT).stream().sorted().collect(Collectors.toList());
```

Comment:
```suggestion
        return randomNumbers.subList(0, GET_NUMBER_COUNT)
                            .stream()
                            .sorted()
                            .collect(Collectors.toList());
```

객체지향 생활 체조 원칙
규칙 4: 한 줄에 점을 하나만 찍는다.

stream 같은 체이닝 메서드를 사용할 때는 해당 규칙을 지켜주면 코드 읽는사람이 더 편한 것 같더라구요 :)

다른쪽도 확인 부탁드립니다~

Reply:
15줄 제한의 압박에 stream 체이닝 메서드를 한줄로 써버렸네요. 😅
관련 코드 수정하고, 다음에는 객체지향 생활 체조 원칙에 부합한 코드 작성을 하도록 하겠습니다.